README.md

```diff
@@ -11,11 +11,6 @@ Developed a project which displays a list of books. The user can add books by pr
 - css/sass
 - Javascript
 
-## Acknowledgements
-
-- Original design idea by [Cindy Shin in Behance.](https://www.behance.net/adagio07)
-- [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/)
-
 ## Authors
 
 - GitHub: [@Edward-Kabue](https://github.com/Edward-Kabue)
```

Comment:
- [ ] Ensure there are commits from the other author.

- [ ] This is a collaborative project, and you are missing the other author in your documentation, kindly add the author details here.